src/main/scala/wdl4s/cwl/CwlEncoders.scala

```diff
@@ -0,0 +1,22 @@
+package wdl4s.cwl
+
+import eu.timepit.refined.api.Refined
+import io.circe.{Encoder, Json}
+import wdl4s.cwl.CwlType.CwlType
+import wdl4s.cwl.CwlVersion.CwlVersion
+import io.circe.syntax.EncoderOps
+
+/**
+  * wdl4s
+  * Created by oruebenacker on 06.07.17.
+  */
+object CwlEncoders {
```

Comment:
Would you please declare these [in package.scala as the decoders are](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/scala/wdl4s/cwl/package.scala#L81)?

e.g. `implicit val scatterMethodEncoder = Encoder.enumEncoder(ScatterMethod)`